posts/2015-05-24-es6-gotchas.md

```diff
@@ -46,6 +46,21 @@ export default class ComponentName extends React.Component {
   }
 ~~~
 
+Another alternative syntax is to use the `::` ES7 [bind](https://github.com/zenparsing/es-function-bind)
+
+~~~js
+export default class ComponentName extends React.Component {
+
+  handleClick(e) => {
+    ...
+  }
+  
+  render(){
+    return (
+      <button onClick={::this.handleClick}> click </button> 
```

Comment:
I think we might be better off putting both this and just calling `.bind(this)` in `render()` under a "things not to do" section. I guess it's more of an "ES7-strawman-gotcha" than an ES6 gotcha, but I can see people making the mistake of using the unary form of the `::` operator in places where it's not ideal.